Modelica/Electrical/Machines/Sensors/VoltageQuasiRMSSensor.mo

```diff
@@ -40,8 +40,7 @@ equation
   connect(ToSpacePhasor1.y, ToPolar1.u) annotation (Line(
       points={{0,-1},{0,-18}}, color={0,0,127}));
   connect(ToPolar1.y[1], Gain1.u) annotation (Line(
-      points={{-1.9984e-15,-41},{-1.9984e-15,-49.5},{0,-49.5},{0,-58}},
-                                                          color={0,0,127}));
+      points={{0,-41},{0,-49.5},{0,-49.5},{0,-58}}, color={0,0,127}));
```

Comment:
Double {0,-49.5},{0,-49.5}. It seems it can be deleted completely because the remaining {{0,-41},{0,-58}} makes the line!

Reply:
Have commited this.